contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--3.0.0--3.1.0.sql

```diff
@@ -964,33 +964,6 @@ CREATE OR REPLACE VIEW information_schema_tsql.SEQUENCES AS
 
 GRANT SELECT ON information_schema_tsql.SEQUENCES TO PUBLIC;
 
-CREATE OR REPLACE VIEW information_schema_tsql.tables AS
```

Comment:
Why are we removing this? We shouldn't change an old upgrade script.

Reply:
First, 3.1 is already released and its stable would have contain expected changes for information_schema_tsql.table so this change would not cause any harm to it. Second, reason to remove is that, we have update the data type of 'TABLE_TYPE' column to sys.varchar in the version 2.7 so if we try to recreate this view here using pg_catalog.varchar then it would result in upgrade failure as we can not change data type of column.

Reply:
@Deepesh125 Do we need this data type change necessarily in 2x? If not, we can avoid this change in older upgrade script which I think is the ideal way.

Reply:
Sounds good.

Reply:
@shalinilohia50 yes, BABEL-4410 is marked for PG14 also